scripts/scil_tractogram_smooth.py

```diff
@@ -2,18 +2,18 @@
 # -*- coding: utf-8 -*-
 
 """
-This script will smooth the streamlines, usually to remove the
-'wiggles' in probabilistic tracking.
-Two choices of methods are available:
-- Gaussian will use the surrounding coordinates for smoothing.
-Streamlines are resampled to 1mm step-size and the smoothing is
-performed on the coordinate array. The sigma will be indicative of the
-number of points surrounding the center points to be used for blurring.
+This script will smooth the streamlines, usually to remove the 'wiggles' in
+probabilistic tracking.
 
-- Spline will fit a spline curve to every streamline using a sigma and
-the number of control points. The sigma represents the allowed distance
-from the control points. The control points for the spline fit will be
-the resampled streamline.
+Two choices of methods are available:
+- Gaussian will use the surrounding coordinates for smoothing. Streamlines are
+resampled to 1mm step-size and the smoothing is performed on the coordinate
+array. The sigma will be indicative of the  number of points surrounding the
+center points to be used for blurring.
+- Spline will fit a spline curve to every streamline using a sigma and the
+number of control points. The sigma represents the allowed distance from the
+control points. The control points for the spline fit will be the resampled
```

Comment:
I know you didn't change this sentence, but looking at the code, it is not the control points that form the resampled streamline. The control points are used to parameterize the spline and then the resampled streamline is obtained by evaluating the spline at constant intervals.